style.css

```diff
@@ -0,0 +1,3 @@
+h2 {
+  color: red;
+}
```

Comment:
- [ OPTIONAL] I noticed that the CSS selector used is `h2` instead of `h1` which is in your **_index.html_** file. The wrong selector means that style declarations will be ineffective. It would be nice to change the selector to `h1`. 👍